### PR TITLE
feat: add a new command for feedback

### PR DESCRIPTION
**Pull request by asalkeld**

### CODE REVIEW

File: pkg/cmd/feedback.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+				Name: "repo",
+				Prompt: &survey.Select{
+					Message: "What is the name of the repo?",
+					Options: []string{"cli", "nitric", "docs", "apis", "node-sdk", "go-sdk"},
+				},
+			},
+			{
+				Name: "kind",
+				Prompt: &survey.Select{
+					Message: "What kind of feedback to you want to give?",
```
Comment (jyecusch) [inline pkg/cmd/feedback.go]: Typo: "to you want to give?' - should be: "_do_ you want to give?"

File: pkg/cmd/feedback.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+			{
+				Name: "title",
+				Prompt: &survey.Input{
+					Message: "How would you like to title your feedback?",
+				},
+			},
+			{
+				Name: "body",
+				Prompt: &survey.Editor{
+					Message:       "Please write you feedback",
```
Comment (jyecusch) [inline pkg/cmd/feedback.go]: Typo: Please write you**r** feedback

**Review by jyecusch — DISMISSED** [review]
LGTM, just a couple typos

File: pkg/cmd/feedback.go (hunk 1 above)
Comment (asalkeld) [reply pkg/cmd/feedback.go]: thanks

File: pkg/cmd/feedback.go (hunk 2 above)
Comment (asalkeld) [reply pkg/cmd/feedback.go]: yikes

**Review by davemooreuws — APPROVED** [review]
Nice

**Review by tjholm — APPROVED** [review]

**Pull request merged**